### ISSUE TITLE
Long league and team names cause notifications validation to fail

### ISSUE THREAD

**Issue by mkrl** [ozfortress/citadel#390]
Whenever a team with a name that is long enough creates a roster for a league with a long name, a notification  "You have been entered in #{league.name} with #{roster.name}." is being created. 

Whenever team name + league name + 32 characters exceeds 128, creation service fails to call `notify_players`.

In one case (approving transfers for teams and leagues fitting the criteria above), this causes a 422 response from the server

**Comment by mkrl** [comment]
I fixed it in my local fork like this:

```
diff --git a/app/services/leagues/rosters/creation_service.rb b/app/services/leagues/rosters/creation_service.rb
index ddfc534..76c6c73 100644
--- a/app/services/leagues/rosters/creation_service.rb
+++ b/app/services/leagues/rosters/creation_service.rb
@@ -19,7 +19,7 @@ module Leagues
       private
 
       def notify_players(roster, league)
-        msg  = "You have been entered in #{league.name} with #{roster.name}."
+        msg  = "You have been entered in #{league.name.truncate(48, separator: ' ')} with #{roster.name.truncate(48, separator: ' ')}."
         link = team_path(roster.team)
 
         roster.players.each do |player|

```

I didn't really want to extend the limit or add additional checks so this just cuts names at a natural break if they are over 48 characters each.

Do you have a better solution in mind or I should make a PR on this?

**Comment by BenjaminSchaaf** [comment]
I think we should remove the limit on notifications, but truncating long names is *also* a good idea. If you could do a PR for both that would be great!

**Comment by BenjaminSchaaf** [comment]
Fixed by fdb6d013f972ac4a5dfacf3c7539568dd6eeb73a
